style.css

```diff
@@ -0,0 +1,39 @@
+
+  
+#main{
+    align-items: center;
+    background-color: rgb(184, 179, 179);
+    display: flex;
+    flex-direction: column ;
+}
+#title{
+    color:black;
+    align-items: center;
+
+}
+ul{
+    font-size:medium;
+    justify-content: center;
+    width:50%;
+    
+}
+li{
+    margin:15px;
+}
+.p{
+    align-items: center;
+    
+}
+.list{
+   align-items: center;
+   display: flex;
+   flex-direction: column;
+}
+p{
+    width: 50%;
+    font-size: 16;
+}
+#img-div{
+    background-color: beige;
+
```

Comment:
It would be better if you give margin and padding zero at the beginning of your CSS as it will make your structure clean. Kindly also add a box-sizing border box which will make the look and feel of the website better.